config/checkstyle-input-suppressions.xml

```diff
@@ -95,8 +95,6 @@
             files="checks[\\/]coding[\\/]packagedeclaration[\\/]InputPackageDeclarationPlain.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]coding[\\/]packagedeclaration[\\/]InputPackageDeclarationWithCommentOnly.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThis15Extensions.java"/>
   <suppress id="UnnecessaryOkComment"
```

Comment:
Everything lgtm....

@raghuramala54 , 
it is not recommended to close and open new PR. All changes can be made through git cmd on existing PR itself  ;)

Reply:
oh thank you for the feedback @MANISH-K-07 , felt overwhelming in the beginning. sorry abt that